bundles/org.eclipse.passage.lic.mail/OSGI-INF/org.eclipse.passage.lic.mail.LicenseMailManagerImpl.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" name="org.eclipse.passage.lic.mail">
+	<implementation class="org.eclipse.passage.lic.internal.mail.LicenseMailManagerImpl"/>
+	<reference bind="bindLicenseMailBuilder" cardinality="0..1" interface="org.eclipse.passage.lic.api.mail.LicenseMailBuilder" name="LicenseMailBuilder" policy="static" unbind="unbindLicenseMailBuilder"/>
+	<service>
+		<provide interface="org.eclipse.passage.lic.mail.LicenseMailManager"/>
```

Comment:
Having interface and its implementation in the same bundle makes impossible to switch implementation